Gaijinizer patch/Map130.txt

```diff
@@ -69,15 +75,16 @@ I will keep this light on until we leave this cave.#Aesthetic
 
  それはまた今度、マイルームでね♡
 > CONTEXT: Map130/events/1/pages/1/35/Dialogue/enemy
-………To be honest, there's a ton of things I want to do
-with you at this size.  
-Let's save it for when you go to my room♡#Aesthetic
+...To be honest, there's a ton of special things
+I want to do that I can only do at this size, but
+let's save them for the next time you go to My
+Room. ♡#Aesthetic/Ron
 > END STRING
```

Comment:
Minor rewording:
>_**...To be honest, there's a ton of special things
I want to do that I can only do like this, but let's
save them for the next time you go to My
Room.**_